common-content/en/module/decomposition/adding-quotes/index.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Adding quotes"
+headless = true
+time = 60
+facilitation = false
+emoji= "➕"
+hide_from_overview = true
+objectives = [
+    "POST data from a frontend to a backend in JSON format.",
+]
++++
+
+{{<note type="Exercise">}}
+Add a form to your frontend which allows users to add quotes to the backend's list of quotes.
+
+Note: Your backend expects the quotes to be submitted as JSON. This means you will need to use a `fetch` request from JavaScript to do the POSTing.
+
+You can't just use a `<form method="POST">` tag because that would post the information in a different format. (It would also redirect the user to a page which just says "ok" after submitting the quote, which isn't a great user experience!)
```

Comment:
I don't think they know what POST is really. Do we want to bring anything in from servers module or no?

Reply:
I agree they probably don't really understand this, but I'm not sure it's a worthwhile diversion, _particularly_ as we're telling them not to do a thing and giving them an example of what to do instead... WDYT?